.github/workflows/wpt.yml

```
@@ -171,7 +171,7 @@ jobs:
           name: ${{ matrix.kind }}-${{ matrix.head }}-wpt-metadata
           path: wpt-metadata/${{ matrix.kind }}/${{ matrix.head }}
       - name: Upload artifacts
-        if: always()
+        if: success() || failure()
```

Let's still upload artifacts, it can be useful even if the run is canceled
```suggestion
        if: always()
```

---

Give a use case and I will revert. 
Usually a run gets cancelled because of 2 reason a superseding was created or the user manually cancelled it. I both seem like you don't need the info and its a waste of resources.